### ISSUE TITLE
migration file missing?

### ISSUE THREAD

**Issue by XaviP** [divio/django-simple-sso#30]
It seems that a migration file is missing in the package (django-simple-sso==0.13.2, django 1.11, py 3.5)
```
python manage.py makemigrations --dry-run
venv/lib/python3.5/site-packages/simple_sso/sso_server/migrations/0003_auto_20180405_0822.py
    - Alter field private_key on consumer
    - Alter field public_key on consumer
    - Alter field access_token on token
    - Alter field request_token on token
```

**Comment by avalanchy** [comment]
Yeah, I have the same issue.
- Python 3.7.0
- Django==2.1
- django-simple-sso==0.14.0

Running `migrate` results with
```
Running migrations:
  No migrations to apply.
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```
Running `makemigrations`:
```
Migrations for 'sso_server':
  venv/lib/python3.7/site-packages/simple_sso/sso_server/migrations/0003_m.py
    - Alter field private_key on consumer
    - Alter field public_key on consumer
    - Alter field access_token on token
    - Alter field request_token on token
```

The fix appears to be removing byte strings from `0001_initial.py`:
```
sed -i "s/b'/'/" backend/venv/lib/python3.7/site-packages/simple_sso/sso_server/migrations/0001_initial.py
```

Any clues on how to bypass this but not by a fork, waiting for a new version or by overwriting Django commands?

**Comment by GaretJax** [comment]
This has been fixed in 0.14.1.
